target/linux/mvebu/files/arch/arm/boot/dts/armada-380-iij-sa-w2.dts

```diff
@@ -91,44 +95,56 @@
 		led-4 {
 			gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
 			color = <LED_COLOR_ID_GREEN>;
+			function = "mobile";
```

Comment:
I do not like it when this is defined as a free text, maybe try to get such a constant into the upstream kernel. an LED for mobile network connection is pretty common.

Reply:
I see. I'll try to send a patch for adding "mobile".